### PR TITLE
Fix/build sql mbr text

### PR DESCRIPTION
**Pull request by irfanshar**

### CODE REVIEW

File: src/scripts/launch [hunk 1]
```diff
@@ -62,6 +63,7 @@ if [ -n "$precmd" ]; then
 fi
 cl $cl_parms "$cmd" 
 exitCode=$?
+cl "$text_command"
```
Comment (edmundreinhardt) [inline src/scripts/launch:66]: Doesn't this assume that the text_command is always set, but it is only set for SQL

**Review by edmundreinhardt — CHANGES_REQUESTED** [review]

File: src/scripts/launch (hunk 1 above)
Comment (edmundreinhardt) [reply src/scripts/launch:66]: Do you have to have to check if text_command is set and assume it is not set if only 8 parms are passed

**Comment by edmundreinhardt** [comment]
We have a problem, the MEMBER text is updated but not the O BJECT text which is what is desired

**Comment by edmundreinhardt** [comment]
> We have a problem, the MEMBER text is updated but not the O BJECT text which is what is desired

image

image

**Comment by edmundreinhardt** [comment]
> > We have a problem, the MEMBER text is updated but not the O BJECT text which is what is desired
> 
 Fixed using CHGOBJD
 
 Now is is necessary to be so far indented for SQL and free RPG?

**Comment by edmundreinhardt** [comment]
I switched to CHGOBJD and updated the bob-recursive-example to test it.  Some objects still not working.

**Comment by irfanshar** [comment]
> > > We have a problem, the MEMBER text is updated but not the O BJECT text which is what is desired
> 
> Fixed using CHGOBJD
> 
> Now is is necessary to be so far indented for SQL and free RPG?

The indentation isn't necessary, but was done to match ARCAD's example https://github.com/aecspades/arcad-example/blob/master/QSQLSRC/ARTIINF.TABLE

**Review by edmundreinhardt — APPROVED** [review]
looks good

**Pull request merged**